bu_passport/lib/components/event_widget.dart

```diff
@@ -1,66 +1,196 @@
+        ),
         child: Stack(
           children: [
-            Image.asset(
-              widget.event.eventPhoto,
-              width: double.infinity, // Use full width
-              fit: BoxFit.cover, // Cover the container with the image
-            ),
-            // Texts overlaid on the image
             Positioned(
-              bottom: 16.0,
-              left: 16.0,
-              right: 16.0,
-              child: Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: [
-                  Text(
-                    widget.event.eventTitle,
-                    style: TextStyle(
-                      fontSize: 18.0,
-                      fontWeight: FontWeight.bold,
-                      color: Colors.white, // Text color on top of the image
+              bottom: sizedBoxHeight,
+              left: edgeInsets,
+              right: edgeInsets,
+              child: Container(
+                padding: EdgeInsets.all(edgeInsets),
+                decoration: BoxDecoration(
+                  color: Colors.transparent, // Make background transparent
+                ),
+                child: Column(
+                  crossAxisAlignment:
+                      CrossAxisAlignment.start, // Align text left
+                  children: [
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: [
+                            Container(
+                              width: screenWidth * 0.65,
+                              child: Text(
+                                widget.event.eventTitle,
+                                style: TextStyle(
+                                  fontSize: 18.0,
+                                  fontWeight: FontWeight.w700,
+                                  color: Colors.white,
+                                ),
+                                maxLines: 2,
+                                overflow: TextOverflow.ellipsis,
+                              ),
+                            ),
+                            SizedBox(height: sizedBoxHeight * 0.5),
+                            Text(
+                              DateFormat.yMMMd()
+                                  .format(widget.event.eventStartTime),
+                              style: TextStyle(
+                                fontSize: 14.0,
+                                fontWeight: FontWeight.w500,
+                                color: Colors.white,
+                              ),
+                            ),
+                          ],
+                        ),
+                        RichText(
+                          text: TextSpan(
+                            // text: '${widget.event.points}',
+                            text: '30',
```

Comment:
Are the points of events just for demonstrating right now? If so,please add a comment saying that "it will be replaced with real value" (for future development purposes).

Reply:
Just added the comment!